Clinical-Genomics/cg
# add return_samples to customer table

This PR add return_samples to customer table, resolved #374 

How to test:
0. install on stage, run SQL script:add-return-samples-to-customer.sql
1.change in admin

Expected outcome:
changed

- [x] tested by @emiliaol 
- [x] deploy approved by @ingkebil 

This is a minor version bump because this feature expands the UI in backwards compatible manner
